content/posts/getting-started/github-pages/index.fr.md

```diff
@@ -5,11 +5,11 @@ menu:
   sidebar:
     name: Déployer dans Github Pages
     identifier: getting-started-github
-    parent: getting-started
+    parent: Getting-started
```

Comment:
Please fix the capitalization. It should be `getting-started` not `Getting-started`. Otherwise, it will not show up under right parent.

Reply:
Idem, no problem. I see that next week. I promise.